src/Hydra/Serializer/PartialCollectionViewNormalizer.php

```diff
@@ -71,12 +78,29 @@ public function normalize($object, $format = null, array $context = [])
             return $data;
         }
 
+        $metadata = isset($context['resource_class']) && null !== $this->resourceMetadataFactory ? $this->resourceMetadataFactory->create($context['resource_class']) : null;
+        $isPaginatedWithCursor = $paginated && null !== $metadata && null !== $cursorPaginationAttribute = $metadata->getCollectionOperationAttribute($context['collection_operation_name'], 'pagination_via_cursor', null, true);
```

Comment:
I find hese lines a bit long but apart from that 👍

Reply:
If I split them, it will be longer (i.e. repeating these `metadata !== null`) 🙊

Reply:
I know

Reply:
Could be extracted into a private method?